SPECS/moby-engine/moby-engine.spec

```diff
@@ -49,7 +49,7 @@ Requires: docker-init
 Requires: iptables
 Requires: libcgroup
 Requires: libseccomp
-Requires: containerd
+Requires: containerd2
```

Comment:
would something like `Requires: /usr/bin/containerd` work so we don't have to force users into choosing a specific containerd version?

Reply:
After a quick test, adding the binary requirement doesn't build the package correctly